upgrade/v070to080/upgrade.go

```diff
@@ -69,37 +67,53 @@ func doInstanceManagerUpgrade(namespace string, lhClient *lhclientset.Clientset)
 	}
 	for _, im := range imList.Items {
 		if im.Spec.Image != "" {
-			continue
-		}
-		im := &im
-		if types.ValidateEngineImageChecksumName(im.Spec.EngineImage) {
-			ei, err := lhClient.LonghornV1beta1().EngineImages(namespace).Get(im.Spec.EngineImage, metav1.GetOptions{})
+			err := upgradeInstanceManagersLabels(&im, lhClient, namespace)
```

Comment:
Consider adding a separate file for 100to101upgrade.go then following the same approach by calling your new UpgradeCRDs after the 070-080 upgrade. see [here](https://github.com/longhorn/longhorn-manager/blob/bc77aeccd90496384e1a84e185a9eb6e7076e6f4/upgrade/upgrade.go#L207)

That way the different upgrade steps stay independent from each other. So on an update from 070-->101 we would first run the 070->080 upgrade code then afterwards run the 101 ugprade code. It add's a little bit more boilerplate, but keeps the update paths separate. Also add comments on why your update is necessary with a link to the github issue. So in the future it's easy to see what the reason for the update where.

Reply:
Will do it. Thank you!